pkgs/desktops/elementary/desktop/gala/plugins-dir.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/meson.build b/meson.build
+index 6b20a60..205699b 100644
+--- a/meson.build
++++ b/meson.build
+@@ -38,7 +38,7 @@ conf.set_quoted('PACKAGE_VERSION', gala_version)
+ conf.set_quoted('DATADIR', data_dir)
+ conf.set_quoted('PKGDATADIR', pkgdata_dir)
+ conf.set_quoted('PKGLIBDIR', pkglib_dir)
+-conf.set_quoted('PLUGINSDIR', plugins_dir)
++conf.set_quoted('PLUGINSDIR',  '/var/run/current-system/sw/lib/gala/plugins')
```

Comment:
I'm not going to do what we're doing for switchboard because additional gala plugins are rare.

That last one that was popular was the picture-in-picture plugin and that's now a part of gala.

Reply:
Either way, it should be

```suggestion
+conf.set_quoted('PLUGINSDIR',  '/run/current-system/sw/lib/gala/plugins')
```